tools/cldr-apps/js/src/esm/cldrTable.js

```diff
@@ -22,6 +22,16 @@ import * as cldrText from "./cldrText.js";
 import * as cldrVote from "./cldrVote.js";
 import * as cldrXPathUtils from "./cldrXpathUtils.js";
 
+const HEADER_ID_PREFIX = "header:";
+
+/*
+ * TODO: usage of '@' in tr.id appears to be problematic for jQuery:
+ * if we try to use selectors like $('#' + tr.id), we get
+ * "Syntax error, unrecognized expression: #r@f3d4397b739b287"
+ * Is there a good reason to keep '@'?
```

Comment:
no, fine to change it

Reply:
I don't think it shows up in user URLs, so should not have an impact.

COlon `:` may need escaping in urls.

Reply:
"Technically, in [HTML5](https://developer.mozilla.org/en-US/docs/Glossary/HTML5), the value for an id attribute may contain any character, except [whitespace](https://developer.mozilla.org/en-US/docs/Glossary/Whitespace) characters. However, to avoid inadvertent errors, only [ASCII](https://developer.mozilla.org/en-US/docs/Glossary/ASCII) letters, digits, '_', and '-' should be used and the value for an id attribute should start with a letter." https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

Reply:
I've revised this PR -- now the prefixes are "header_" and "row_", and all non-alphanumeric characters are replaced by underscore in the id